2025-02-06 22:12:51,307 INFO    Thread-48 :662016 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2025-02-06 22:12:51,308 INFO    Thread-48 :662016 [wandb_setup.py:_flush():79] Configure stats pid to 662016
2025-02-06 22:12:51,308 INFO    Thread-48 :662016 [wandb_setup.py:_flush():79] Loading settings from C:\Users\admin\.config\wandb\settings
2025-02-06 22:12:51,308 INFO    Thread-48 :662016 [wandb_setup.py:_flush():79] Loading settings from C:\Users\admin\Documents\GitHub\pum-24\wandb\settings
2025-02-06 22:12:51,308 INFO    Thread-48 :662016 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'project': 'mnist-classifier', 'entity': 'tiopirfur-uniwersytet-jagiello-ski-w-krakowie', 'root_dir': 'C:\\Users\\admin\\Documents\\GitHub\\pum-24', 'run_id': 'tmp4qdws', 'sweep_param_path': 'C:\\Users\\admin\\Documents\\GitHub\\pum-24\\wandb\\sweep-c9y587ur\\config-tmp4qdws.yaml', 'sweep_id': 'c9y587ur'}
2025-02-06 22:12:51,309 INFO    Thread-48 :662016 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2025-02-06 22:12:51,309 INFO    Thread-48 :662016 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-02-06 22:12:51,309 INFO    Thread-48 :662016 [wandb_setup.py:_flush():79] Applying login settings: {}
2025-02-06 22:12:51,310 INFO    Thread-48 :662016 [wandb_init.py:_log_setup():533] Logging user logs to C:\Users\admin\Documents\GitHub\pum-24\wandb\run-20250206_221251-tmp4qdws\logs\debug.log
2025-02-06 22:12:51,311 INFO    Thread-48 :662016 [wandb_init.py:_log_setup():534] Logging internal logs to C:\Users\admin\Documents\GitHub\pum-24\wandb\run-20250206_221251-tmp4qdws\logs\debug-internal.log
2025-02-06 22:12:51,311 INFO    Thread-48 :662016 [wandb_init.py:_jupyter_setup():479] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001F9041439D0>
2025-02-06 22:12:51,312 INFO    Thread-48 :662016 [wandb_init.py:init():619] calling init triggers
2025-02-06 22:12:51,312 INFO    Thread-48 :662016 [wandb_init.py:init():626] wandb.init called with sweep_config: {'batch_size': 128, 'layer_1_size': 256, 'layer_2_size': 128, 'learning_rate': 0.01}
config: {}
2025-02-06 22:12:51,313 INFO    Thread-48 :662016 [wandb_init.py:init():669] starting backend
2025-02-06 22:12:51,313 INFO    Thread-48 :662016 [wandb_init.py:init():673] sending inform_init request
2025-02-06 22:12:51,351 INFO    Thread-48 :662016 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn, using: spawn
2025-02-06 22:12:51,352 INFO    Thread-48 :662016 [wandb_init.py:init():686] backend started and connected
2025-02-06 22:12:51,421 INFO    Thread-48 :662016 [wandb_run.py:_config_callback():1387] config_cb None None {'batch_size': 128, 'layer_1_size': 256, 'layer_2_size': 128, 'learning_rate': 0.01}
2025-02-06 22:12:51,423 INFO    Thread-48 :662016 [wandb_run.py:_label_probe_notebook():1339] probe notebook
2025-02-06 22:12:51,430 INFO    Thread-48 :662016 [wandb_init.py:init():781] updated telemetry
2025-02-06 22:12:51,431 INFO    Thread-48 :662016 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2025-02-06 22:12:51,826 INFO    Thread-48 :662016 [wandb_init.py:init():867] starting run threads in backend
2025-02-06 22:12:53,106 INFO    Thread-48 :662016 [wandb_run.py:_console_start():2456] atexit reg
2025-02-06 22:12:53,106 INFO    Thread-48 :662016 [wandb_run.py:_redirect():2305] redirect: wrap_raw
2025-02-06 22:12:53,106 INFO    Thread-48 :662016 [wandb_run.py:_redirect():2370] Wrapping output streams.
2025-02-06 22:12:53,107 INFO    Thread-48 :662016 [wandb_run.py:_redirect():2395] Redirects installed.
2025-02-06 22:12:53,109 INFO    Thread-48 :662016 [wandb_init.py:init():911] run started, returning control to user process
2025-02-06 22:13:03,432 INFO    Thread-48 :662016 [wandb_run.py:_finish():2155] finishing run tiopirfur-uniwersytet-jagiello-ski-w-krakowie/mnist-classifier/tmp4qdws
2025-02-06 22:13:03,433 INFO    Thread-48 :662016 [jupyter.py:save_history():459] not saving jupyter history
2025-02-06 22:13:03,434 INFO    Thread-48 :662016 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2025-02-06 22:13:03,434 INFO    Thread-48 :662016 [wandb_init.py:_jupyter_teardown():461] cleaning up jupyter logic
2025-02-06 22:13:03,434 INFO    Thread-48 :662016 [wandb_run.py:_atexit_cleanup():2420] got exitcode: 1
2025-02-06 22:13:03,434 INFO    Thread-48 :662016 [wandb_run.py:_restore():2402] restore
2025-02-06 22:13:03,435 INFO    Thread-48 :662016 [wandb_run.py:_restore():2408] restore done
2025-02-06 22:13:05,428 INFO    Thread-48 :662016 [wandb_run.py:_footer_history_summary_info():3960] rendering history
2025-02-06 22:13:05,428 INFO    Thread-48 :662016 [wandb_run.py:_footer_history_summary_info():3992] rendering summary
2025-02-06 22:13:05,453 INFO    Thread-48 :662016 [wandb_run.py:_footer_sync_info():3921] logging synced files
